kratos/python_scripts/analysis_stage.py

```diff
@@ -2,6 +2,10 @@
 
 # Importing Kratos
 import KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
```

Comment:
Please do not change this class before discussing it!

Btw I think the changes you are doing are just fo debugging, the intended way for doing this for this would be to derive from `StructuralMechanicsAnalysis` and add your custom things